testing.go

```diff
@@ -20,6 +20,12 @@ var errAlreadyInProgress = regexp.MustCompile(`removal of container .* is alread
 // This is a function designed to be used in your test, when Docker is not mandatory for CI/CD.
 // In this way tests that depend on Testcontainers won't run if the provider is provisioned correctly.
 func SkipIfProviderIsNotHealthy(t *testing.T) {
+	defer func() {
```

Comment:
I was not sure if t.Helper should go before the defer block, but it seems so.

Could you please update it, as golangci-lint suggests?

Reply:
Whoops, sorry about that.  I didn't run `golangci-lint` in my haste.

Fixed.